homepage/models.py

```diff
@@ -125,10 +184,54 @@ def get_elective_courses(courses):
         # gets all courses that are electives
         return courses.filter(mandatory=False)
 
-    # --- returns if course has Prerequisites
-    def has_preqs(self):
-        return Prerequisites.does_course_have_prerequisites(self)
+    @staticmethod
+    def get_courses_with_preqs(courses):
+        # gets all courses that have prerequisites

```

Comment:
I think this comment can be removed, its clear from the method's name what it does

Reply:
As preqs is unclear IMO by itself, I made the comment.
However, I don't want to enter the word prerequisites into the method name because it's hard to spell and opens up potential for bugs by misspellings.